persistence/src/main/java/de/cyface/persistence/serialization/MetaFile.java

```diff
@@ -0,0 +1,216 @@
+package de.cyface.persistence.serialization;
+
+import static de.cyface.persistence.Constants.TAG;
+import static de.cyface.persistence.serialization.MeasurementSerializer.PERSISTENCE_FILE_FORMAT_VERSION;
+
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+import android.content.Context;
+import android.util.Log;
+
+import de.cyface.persistence.FileUtils;
+import de.cyface.persistence.model.Vehicle;
+
+/**
+ * The file format to persist meta information of a measurement.
+ *
+ * @author Armin Schnabel
+ * @version 1.0.0
+ * @since 3.0.0
+ */
+public class MetaFile implements FileSupport<MetaFile.PointMetaData> {
+
+    /**
+     * The {@link File} pointer to the actual file.
+     */
+    private final File file;
+    /**
+     * The name of the file containing the data
+     */
+    public static final String FILE_NAME = "m";
+    /**
+     * The name of the file containing the data
+     */
+    public static final String FILE_EXTENSION = "cyfm";
+
+    /**
+     * @param context The {@link Context} required to access the persistence layer.
+     * @param measurementId The identifier of the measurement
+     * @param vehicle The {@link Vehicle} used in the measurement
+     */
+    public MetaFile(final Context context, final long measurementId, final Vehicle vehicle) {
+        this.file = new FileUtils(context).createFile(measurementId, FILE_NAME, FILE_EXTENSION);
+        // In case the PointMetaData is not persisted as the end of the capturing, appending the
+        // vehicle at the beginning of the capturing allows to restore the corrupted data later on
+        write(vehicle);
+    }
+
+    @Override
+    public void append(final PointMetaData metaData) {
+        final byte[] data = serialize(metaData);
+        append(data, this.file);
+    }
+
+    private static void append(final byte[] data, final File file) {
+        try {
+            final BufferedOutputStream outputStream = new BufferedOutputStream(new FileOutputStream(file, true));
+            Log.d(TAG, "Writing " + data.length + " bytes (4 counters)");
+            outputStream.write(data);
+            outputStream.close();
```

Comment:
This results in a memory leak if the output stream fails while writing the data. Use finally!